cmd/minishift/cmd/start.go

```diff
@@ -230,6 +230,12 @@ func runStart(cmd *cobra.Command, args []string) {
 		postClusterUp(hostVm, clusterUpConfig)
 		exportContainerImages(hostVm, requestedOpenShiftVersion)
 	}
+	if isRestart {
+		err = cmdUtil.SetOcContext(profileActions.GetActiveProfile())
+		if err != nil {
+			fmt.Println(fmt.Sprintf("Could switch oc CLI context for: %s", profileActions.GetActiveProfile()))
+		}
+	}
```

Comment:
This is wrong. As user might have started the start with `minishift --profile foo start`. In that case `active` profile is different  than constants.ProfileName. You should use `constants.ProfileName` as it would have the profile name you are interested.

Reply:
s/Could switch/Could not switch/

Reply:
probably "could not set or activate"